cmd/storage/wipe.go

```diff
@@ -35,28 +31,31 @@ func WipeDisks() error {
 
 	log.Info("wipe existing disks", "disks", disks)
 
-	wipeErrors := make(chan error)
-	var wg sync.WaitGroup
-	wg.Add(len(disks))
+	g, _ := errgroup.WithContext(context.Background())
 	for _, disk := range disks {
-		go func(disk *ghw.Disk) {
-			defer wg.Done()
+		disk := disk
+		properties, err := FetchBlockIDProperties(fmt.Sprintf("/dev/%s", disk.Name))
+		if err != nil {
+			log.Warn("failed to detect disk properties", "error", err)
+		}
+		disktype, ok := properties["TYPE"]
+		if ok && strings.Contains(disktype, "isw_raid") {
+			log.Info("skip raid member", "disk", disk.Name)
+			continue
+		}
+
+		g.Go(func() error {
 			if strings.HasPrefix(disk.Name, DiskPrefixToIgnore) {
-				return
+				return nil
 			}
```

Comment:
This can also be moved outside the go routine like the other skip condition.

Reply:
Maybe also emit a log message that it skipped because its prefix is ignored for wipe.